src/enums/Algorithm.java

```diff
@@ -0,0 +1,43 @@
+package enums;
+
+/**
+ * All the algorithms that can be used for signing or verifying a token as determined by Auth0
+ */
+public enum Algorithm {
```

Comment:
please make sure that Leo and you are sharing this class.
@lccodes FYI

Reply:
@lyzhang27